### PR TITLE
0411 function

### PR DESCRIPTION
**Pull request by bintabaldeh**

### CODE REVIEW

**Comment by bintabaldeh** [comment]
how about this one

File: 0411.scm [hunk 1]
```diff
@@ -0,0 +1,52 @@
+(define rember*
+  (lambda (a l)
+    (cond
+      ((null? l) (quote ()))
+      ((atom? (car l))
+        (cond
+          ((eq? (car l) a)
+            (rember* a (cdr l)))
+            (else (cons (car l)
```
Comment (hrl7) [inline 0411.scm]: fix this indent

File: 0411.scm [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              (rember* a(cdr l))))))
+              (else (cons (rember* a(car l))
```
Comment (hrl7) [inline 0411.scm]: also here

File: 0411.scm [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+(define atom?
+  (lambda (x)
+  (and (not (null? x)) (not (pair? x)))))
+
+(print (rember* 'apple '(apple banana orange)))
+(print (rember* 'apple '(apple banana apple)))
+(print (rember* 'apple '(apple (banana apple))))
+
+(define rm-even*
+  (lambda (lat)
```
Comment (hrl7) [inline 0411.scm:22]: even it is a "star" function, is this `lat`? really?

File: 0411.scm [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        (cond
+          ((eq? (car l) a)
+            (rember* a (cdr l)))
+            (else (cons (car l)
+              (rember* a(cdr l))))))
+              (else (cons (rember* a(car l))
+              (rember* a (cdr l)))))))
+(define atom?
+  (lambda (x)
+  (and (not (null? x)) (not (pair? x)))))
```
Comment (hrl7) [inline 0411.scm:15]: this `and` is inside of lambda. so you need one more indent

File: 0411.scm [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+(print (rember* 'apple '(apple banana apple)))
+(print (rember* 'apple '(apple (banana apple))))
+
+(define rm-even*
+  (lambda (lat)
+    (cond
+      ((null? lat) (quote ()))
+      ((even? (car lat)) (rm-even* (cdr lat)))
+      (else (cons (car lat)
+              (rm-even* (cdr lat)))))))
```
Comment (hrl7) [inline 0411.scm:27]: in this function, the indentations are good! :+1:

File: 0411.scm [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+(print (rm-even* '(2 4 6 8 10)))
+(print (rm-even* '(1 2 3 4 5)))
+
+(print (rm-even* '(1 2 3 4)))
+(print (rm-even* '(1 2 3 4)))
+(print (rm-even* '(2 4 6 8)))
+
+(define sum-tup*
```
Comment (hrl7) [inline 0411.scm:35]: it's not a "star" function.
if this is named `sum-tup`, you are right.
now I recommend to change the name to `sum-tup`. 
later let's rewrite this.

File: 0411.scm [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+      ((null? tup) 0)
+      (else (+ (car tup) (sum-tup*( cdr tup)))))))
+
+(print (sum-tup* '(1 2 3 4)))
+(print (sum-tup* '(12 32 48 6)))
+
+(define tup-sum
+  (lambda (tup)
+    (cond
+    ((null? tup) 0)
```
Comment (hrl7) [inline 0411.scm]: the indent is wrong

File: 0411.scm [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    (else (+ (car tup) (tup-sum (cdr tup)))))))
+(print (tup-sum '(1 2 3 4)))
```
Comment (hrl7) [inline 0411.scm]: I need one more empty line above since we want to separate the definition and others.

**Review by hrl7 — CHANGES_REQUESTED** [review]

**Review by hrl7 — APPROVED** [review]

**Pull request merged**